pkgs/development/libraries/uid_wrapper/default.nix

```
@@ -12,9 +12,10 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ cmake pkg-config ];
 
   meta = with lib; {
-    description = "A wrapper for the user, group and hosts NSS API";
+    description = "A testing tool to fake privilege separation";
     homepage = "https://git.samba.org/?p=uid_wrapper.git;a=summary;";
     license = licenses.bsd3;
     platforms = platforms.all;
+    changelog = "https://git.samba.org/?p=uid_wrapper.git;a=blob;f=CHANGELOG;hb=HEAD";
```

```suggestion
    changelog = "https://git.samba.org/?p=uid_wrapper.git;a=blob;f=CHANGELOG;hb=refs/tags/uid_wrapper-${version}";
```
We should link the changelog of the specific commit

---

I didn't wrote this out explicit but I think all the other meta.changelog changes should get the same change.

---

Feel free to fix it - in a similar amount of time compared to the comments written about this so far :)

I intended to do a quick fix as I noticed that package descriptions were copied wrongly. The fact that this went unnoticed since the packages introduction in nixpkgs 9 years ago makes me question how urgent it is to link to individual releases for the changelogs. ;)